test/verify/check-machines-dbus

```diff
@@ -39,6 +39,33 @@ TEST_NETWORK_XML = """
     </network>
 """
 
+TEST_NETWORK2_XML = """
+    <network>
+      <name>test_network2</name>
+      <uuid>5f61aa2a-4e51-400b-a747-546a8488730d</uuid>
```

Comment:
Isn't the uuid generated if you don't define it?

Reply:
Also it's indented without reason.

Reply:
Test please another network without ips. Example
```
<network>
  <name>host-bridge</name>
  <forward mode="bridge"/>
  <bridge name="br0"/>
</network>
```

Reply:
Yes UUID can be omitted.

Reply:
About intendation, there is another XML network example right above mine which is used for different test, and that example is intended. So i would look ugly if one of them would be intended and the other one wasn't.